README.md

```diff
@@ -44,10 +40,25 @@ every time, add the following alias to your `~/.bashrc`:
 alias fluo='fluo-dev fluo'
 ```
 
+Next, you need to download the binary tarballs of Fluo dependencies (i.e Accumulo, Hadoop, and
+Zookeeper).  This can be done by running the command below which downloads all tarballs and their
+corresponding hashes and signatures to the directory specified by DOWNLOADS in your env.sh. It 
+will use Apache download mirror specified by APACHE_MIRROR in env.sh.  Other mirrors can
```

Comment:
Did you think about making these vars from env fixed width like `APACHE_MIRROR`?